Added procedure to create multiple heads

kiryteo opened the pull request:
The steps to get the multi-head network, based on the files to be modified.

wolny reviewed (commented):
Hi Ashwin,

thanks a lot for the detailed recipe of how to use a multi-head network. Although it's very clear it's quite low level and specific to your task. I'd believe it would be of great help if you could abstract away all of the changes (possibly only adding additional `Dataset` class and a new multi-head model class) and provide an example of how to use it. I'm happy to help you with that, since I'm going to make a small refactor (simplify) the repository a bit. Then we could put it as another section in the main readme.

kiryteo commented:
Hi Adrian, thanks for your input! I'll make this more generalized and remove the case-specific details (maybe provide it as an example).

kiryteo commented:
@wolny I've added the multihead dataset class (based on hdf5 class) and modified the multihead routine readme - removing the Kinetochores use case details, its more general now. Please take a look.
